Makefile

```diff
@@ -52,6 +52,10 @@ pre-build:
 run:
 	@cd $(SRVFOLDER) && go run main.go
 
+.PHONY: test
+test:
+	go test ./...
```

Comment:
come gestiamo i test? ha senso averne ora?

Reply:
Assolutamente si, TDD!

Reply:
I'll create a new PR in the next days showing how to do table tests on this handler

Reply:
Sounds good!